cmd/tsbs_load_clickhouse/main.go

```diff
@@ -23,6 +23,7 @@ const (
 // Program option vars:
 var (
 	host     string
+	port     string
```

Comment:
this seems like it should be an `int32` or `int64` and not a string? seems too risky to allow a string where somebody could inject all sorts of weirdness into the constructed connection URIs